data/mods/My_Sweet_Cataclysm/sweet_mutations.json

```diff
@@ -15,7 +15,9 @@
           { "value": "FAT_TO_MAX_HP", "multiply": -0.9 },
           { "value": "HEALTHY_RATE", "multiply": -1 },
           { "value": "SWEAT_MULTIPLIER", "multiply": -1 },
-          { "value": "THIRST", "multiply": -1 }
+          { "value": "THIRST", "multiply": -1 },
+          { "value": "REGEN_HP", "multiply": -1 },
```

Comment:
Shouldn't it be 0?

Reply:
nop, in enchantments `multiply` means `pick value, add 1, multiply on it`; making it zero would be multiplication by one, so it does nothing
` x *= 1 + (-1)` vs `x *= 1 + 0`

Reply:
Weird but ok